extensions/3DTILES_content_gltf/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,33 @@
+# Conversion of Existing Tile Formats
+
+This section covers the differences between existing tile formats and the new glTF approach to tile content, and describes how existing tile formats can be converted into equivalent glTF content.
+
+### Batched 3D Model (b3dm)
+
+[Batched 3D Model](../../specification/TileFormats/Batched3DModel) is a wrapper around a binary glTF that includes additional information in its Feature Table and Batch Table. Batched 3D Model content can be converted into glTF content with the following changes: 
+
+* The `RTC_CENTER` can be added to the translation component of the root node of the glTF asset.
```

Comment:
Maybe link to the section where `RTC_CENTER` is defined? https://github.com/CesiumGS/3d-tiles/tree/main/specification/TileFormats/Batched3DModel#coordinate-system

Reply:
There are some places where I thought that a link to the corresponding documentation could be helpful - and that literally (and figuratively) starts at the highest level, meaning that the first sentence could be "Written against the `[3D Tiles 1.0 specification](link)`." 

If there is a geenral guideline like "Add links when a reference is mentioned for the first time", then I can take that into account in future review passes.